src/matches/matches.service.ts

```diff
@@ -99,4 +108,29 @@ export class MatchesService {
     await this.matchRepository.save(match);
     return match;
   }
+
+  async sendSMS(dtoDate: Date, messageKey: String, phoneNumber: String){
```

Comment:
Add return type (Promise<string>). If this is _async_ then each call of this method must also be placed within _async_ method and must have an _await_ keyword before function call (to resolve Promise). I think in Python "Future" is an equivalent to JS's "Promise".

Reply:
Also add return type